### PR TITLE
Pull in file_manager from curation_concerns

### PR DESCRIPTION
**Pull request by hackmastera**
Please note the javascript behaves wonky with child works. hope to address that in a PR to follow. also things will be somewhat improved when https://github.com/projecthydra/curation_concerns/pull/918 comes through.

I should probably squash these! but def. wondering if it looks like this test could be DRYed up a little. I tried a few things that didn't really work.

### CODE REVIEW

File: app/assets/stylesheets/sufia/_sufia.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 @import 'hydra-editor/multi_value_fields';
 @import 'curation_concerns/modules/forms';
+@import 'curation_concerns/modules/file_manager';
```
Comment (hackmastera) [inline app/assets/stylesheets/sufia/_sufia.scss]: idk houndbot; it doesn't look like it to me

Comment (mjgiarlo) [inline app/assets/stylesheets/sufia/_sufia.scss]: feel free to disregard hound here.

File: spec/views/curation_concerns/base/_show_actions.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  context "as an editor" do
+    before do
+      allow(presenter).to receive(:editor?).and_return(true)
+    end
+    context "when the work does not contain children" do
+      before do
+        allow(presenter).to receive(:member_presenters).and_return([])
+        render 'curation_concerns/base/show_actions.html.erb', presenter: presenter
+      end
+      it "shows file manager link" do
```
Comment (mjgiarlo) [inline spec/views/curation_concerns/base/_show_actions.html.erb_spec.rb]: `s/shows/does not show/`

File: spec/views/curation_concerns/base/_show_actions.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        expect(rendered).to have_link 'Edit'
+        expect(rendered).to have_link 'Delete'
+      end
+    end
+    context "when the work contains children" do
+      before do
+        allow(presenter).to receive(:member_presenters).and_return([member])
+        render 'curation_concerns/base/show_actions.html.erb', presenter: presenter
+      end
+      before { allow(presenter).to receive(:member_presenters).and_return([member]) }
```
Comment (mjgiarlo) [inline spec/views/curation_concerns/base/_show_actions.html.erb_spec.rb]: Looks like cruft. :boom:

**Pull request merged**